superset/views/core.py

```diff
@@ -2638,11 +2638,15 @@ def validate_sql_json(self):
         except Exception as e:
             logging.exception(e)
             msg = _(
-                f"{validator.name} was unable to check your query.\nPlease "
-                "make sure that any services it depends on are available\n"
+                f"{validator.name} was unable to check your query.\n"
+                "Please recheck your query.\n"
```

Comment:
Do we know why the validator is unable to check the query? Is this is something the user can fix? Seems to me that there's nothing the user can do, and the validator should be doing the work of checking the query, no?

Reply:
It is hard to say. The examples I saw were actually checked by the validator and this PR addresses those cases. So I hope that this will catch issues with our code instead of the validator because you are right that there isn't anything the user can do. I was going to check the logs once this PR was checked in and deployed to see if there were other cases I missed.